src/models/timer.c

```diff
@@ -57,6 +57,16 @@ void updateTimers()
     }
 }
 
+void setTimerInitialCount(struct Timer *timer, int duration)
```

Comment:
`setDuration()`

Reply:
so we should change the struct of the timer from initial count to duration ?

Reply:
only the function name change

initialCount is an internal name, but it change the timer duration

Reply:
Alright I will make the method name change tomorrow, and merge this monumental PR ! Thanks for your reviews !

Reply:
Have a great night !!!!